### ISSUE TITLE
Multi-branch integration branches should be automatically used

### ISSUE THREAD

**Issue by mdekrey** [mdekrey/GitAutomation#127]
Sometimes, integration branches apply to several branches. The system doesn't currently support them.

### Scenario
1. Create branches A, B, C
2. Create an integration branch for A-B-C
3. Make an RC containing A, B, C

### Expected
The RC should also receive the integration branch automatically, even if no conflicts are detected.

**Comment by mdekrey** [comment]
Ended up creating endless snakes of integration branches in certain scenarios, like this:

```
  a--------b--------------------------------------
 / \        \         \                           \
o   merge----merge-----merge-----------------------rc
 \ /        /         /                           /
  c-----------------------------------------------
```

**Comment by mdekrey** [comment]
This was solved and released in 0.9.0.
